### PR TITLE
Suggest bc breaks

### LINKED ISSUE #180: BC Breaking changes
- Money::getAmount() returns a string instead of an int value
- stringToUnits has been moved to a parser

### PR DESCRIPTION
**Pull request by frederikbosch**
Fixes #180

### CODE REVIEW

**Comment by sagikazarmark** [comment]
Actually I think it would be better to keep the usual keepachangelog style AND:

- add the `[BC]` tag before each change which is BC breaking
- optionally make it bold
- also a big fat warning that most of the library is rewritten, so apart from the big breaks, there might be minor ones (refer to docs plz)

**Comment by frederikbosch** [comment]
Good idea, will change it.

**Comment by elstamey** [comment]
Thank you!  I'm so glad that users can more readily see breaking changes in the changelog.  We had to find the change to GetAmount's returning a string by reading the code because nothing was obvious in the changelog.

**Comment by frederikbosch** [comment]
@sagikazarmark The PR is updated to your suggestions. My suggestion would be to repeat the BC breaks in the changelog when we release 3.0.0.

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,9 @@
 
 ### Changed
 
+- **[BC break]** Money::getAmount() returns a string instead of an int value
+- **[BC break]** Moved stringToUnits to StringToUnitsParser parser
+- Announced deprecations are removed (Currency::getName, CurrencyPair::getRatio, Money::getUnits)
```
Comment (sagikazarmark) [inline CHANGELOG.md]: This shoulg go into the Removed section IMO

**Comment by sagikazarmark** [comment]
> My suggestion would be to repeat the BC breaks in the changelog when we release 3.0.0.

Sounds like a sensible idea. :+1:

**Comment by frederikbosch** [comment]
@sagikazarmark Updated. I qualified all removed features as BC breaks. Because any removed feature is breaking, right?

**Pull request merged**